src/api.raml

```diff
@@ -497,6 +497,13 @@ traits:
         is:
           - paged: { resources: Collections }
           - subjected: { resources: Collections }
+        queryParameters:
+            containing:
+                description: |
+                    Only return collections containing at least one of these items.
+                type: string[]
+                required: false
+                example: [article/1234, interview/5678]
```

Comment:
I think I over-thought this (when I recognised https://github.com/elifesciences/api-client-php/pull/67 it simplified things, as the slashes don't have to be percent-encoded).

Since PRs are already open, I'm happy to stick with slashes here. Agree than 1 vs many is path vs query string parameters.